cmd/zpool/zpool_main.c

```diff
@@ -2177,7 +2177,8 @@ show_import(nvlist_t *config)
 			break;
 		case ZPOOL_STATUS_HOSTID_REQUIRED:
 			(void) printf(gettext(" action: Set a unique system "
-			    "hostid with the genhostid(1) command.\n"));
+			    "hostid with the genhostid(1) or zgenhostid(8) "
```

Comment:
Since you are providing the zgenhostid(8) command, you might consider to just reference that exclusively, and install it for all distros?